.github/workflows/detect-changed-modules-and-build-reports.yml

```diff
@@ -13,10 +13,12 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 1000
+      - name: Set python-version
+        run: echo "PYTHON_VERSION="$(cat ../../.python-version)"" >> $GITHUB_ENV
```

Comment:
This relative path might just need to be `.python-version`. Not sure what directory github workflows run out of